_layouts/test.html

```diff
@@ -1,13 +1,112 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
```

Comment:
Don’t close void elements

(May affect other pages, too)

Reply:
this is coming when i save it's automatic generated by the prettier, 
it is still valid
https://stackoverflow.com/questions/19506028/do-you-need-to-close-meta-and-link-tags-in-html

Reply:
It’s valid, yes, but it’s unnecessary. Let’s update the config for Prettier then—doesn’t need to happen now, unless Prettier would enforce this code.

Reply:
I saved without formatting to be able to merge this fast, will try to remove it